islandora_videojs.module

```diff
@@ -40,7 +58,9 @@ function islandora_videojs_preprocess_islandora_videojs(array &$variables) {
   $videojs_path = libraries_get_path("video-js");
   drupal_add_js($videojs_path . "/video.js");
   drupal_add_css($videojs_path . '/video-js.css');
-
+  if (variable_get('hls_library') === 1) {
```

Comment:
Should likely default the `variable_get` to `FALSE` and check the equivalency with a `== FALSE`.